### ISSUE TITLE
What is meaning of `qrel = 2` and `qrel = 3` in the test dataset of qrel?

### ISSUE THREAD

**Issue by rossbrown9879** [microsoft/MSMARCODocumentRanking#1]
In the MSMARCO document ranking dataset, in trec 2019 challenge, the qrel file for test set contains column called rating, which has qrel values equal to 2, 3 and so on. As far as I know, the qrel file is dataset of queries and their relevant documents, I thought if qrel value of document for some query is 0, then its not relevant and if it is 1, then the document is relevant. 

How can I interpret this rating in the qrel test dataset file? This is a link to the dataset  : https://trec.nist.gov/data/deep/2019qrels-docs.txt

**Comment by bmitra-msft** [comment]
The TREC Deep Learning track training data (which is based on the original MS MARCO dataset) has binary labels. The test sets however were composed by judging all the submitted runs from last year's track. The TREC judgments are on a five point scale. This means there is some distributional difference between the training and test data but to be honest this is a common scenario---because labeling is costly, the training data in IR often use automatic labels based on user behavior data or other means. One way to look at this is through the lens of domain adaptation. In practice, most runs last year ignored this difference.

Lastly, if your question was more about evaluation metrics then computing NDCG and NCG using the graded test labels would be the recommended way to go.

**Comment by bmitra-msft** [comment]
Btw, the exact meaning of the grades is described in the overview paper (see Section 2).
https://arxiv.org/pdf/2003.07820.pdf
